### PR TITLE
feat: refactor home page

### PR DESCRIPTION
**Pull request by MaloPolese**
Closes # 

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code is documented
- [ ] I provided unitary tests or procedure to test my code
- [x] My PR have a clear description of what my code is supposed to do

### CODE REVIEW

**Comment by LeoFVO** [comment]
@MaloPolese Can you lint your code please

**Comment by LeoFVO** [comment]
> @MaloPolese Can you lint your code please

Can you also rebase your branch to remove merge commit :)

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <Paper
+      variant="outlined"
+      component={'footer'}
+      sx={{
+        minHeight: '200px',
+        p: 2,
+        mt: 4,
+      }}
+    >
+      {/* TODO */}
```
Comment (LeoFVO) [inline apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:15]: No footer for the moment ?

File: apps/polyflix/src/modules/core/components/ListVideoCard/list-video-card.component.tsx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        )}
+        {/* Uncomment this line to show video duration */}
+        {/* <ElementSmallTag
+          sx={{
+            position: 'absolute',
+            right: 4,
+            bottom: 4,
+          }}
+          text="20 min"
+        /> */}
```
Comment (LeoFVO) [inline apps/polyflix/src/modules/core/components/ListVideoCard/list-video-card.component.tsx:102]: Why this ?

File: apps/polyflix/src/modules/core/components/VideoCard/video-card.component.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+        )}
+        {/* Uncomment this line to show video duration */}
+        {/* <ElementSmallTag
+          sx={{
+            position: 'absolute',
+            right: 8,
+            bottom: 8,
+          }}
+          text="20 min"
+        /> */}
```
Comment (LeoFVO) [inline apps/polyflix/src/modules/core/components/VideoCard/video-card.component.tsx:84]: same

File: apps/polyflix/src/modules/core/layouts/Dashboard/Dashboard.style.tsx [hunk 4]
```diff
@@ -12,16 +12,16 @@ export const RootStyle = styled('div')({
...
+  // paddingBottom: theme.spacing(10),
   display: 'flex',
   flexDirection: 'column',
   alignItems: 'center',
-  [theme.breakpoints.up('lg')]: {
-    paddingTop: APP_BAR_DESKTOP + 24,
-  },
+  // [theme.breakpoints.up('lg')]: {
+  //   paddingTop: APP_BAR_DESKTOP + 24,
+  // },
```
Comment (LeoFVO) [inline apps/polyflix/src/modules/core/layouts/Dashboard/Dashboard.style.tsx:26]: Are this comment useful or just a misstake

File: apps/polyflix/src/modules/users/components/UserAvatar/UserAvatar.component.tsx [hunk 5]
```diff
@@ -1,23 +1,28 @@
...
   return (
     <Avatar
       sx={{ ...props.sx }}
       variant={props.variant}
-      src={props.user?.avatar}
-      // alt={`${props.user?.displayName} profile picture`}
-      alt={`${props.user?.firstName} ${props.user?.lastName} profile picture`}
+      src={user?.avatar}
+      // alt={`${user?.displayName} profile picture`}
+      alt={`${user?.firstName} ${user?.lastName} profile picture`}
```
Comment (LeoFVO) [inline apps/polyflix/src/modules/users/components/UserAvatar/UserAvatar.component.tsx:25]: Prefer using user?.username

**Review by LeoFVO — CHANGES_REQUESTED** [review]

File: apps/polyflix/src/modules/core/components/Jumbotron/jumbotron.component.tsx [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+                      alpha(theme.palette.common.black, 0.7),
+                    px: 2,
+                    py: 0.5,
+                    borderRadius: 1,
+                  }}
+                >
+                  {element.title}
+                </Typography>
+              </Box>
+            </SwiperSlide>
```
Comment (LeoFVO) [inline apps/polyflix/src/modules/core/components/Jumbotron/jumbotron.component.tsx:151]: It's seems that we cannot click on the carousel slide to jump to the video, right ?

**Comment by LeoFVO** [comment]
When we are on the homepage, if we scroll down the searchbar isn't reachable anymore and any alternative is available to search content..

File: apps/polyflix/src/modules/core/components/ListVideoCard/list-video-card.component.tsx (hunk 2 above)
Comment (MaloPolese) [reply apps/polyflix/src/modules/core/components/ListVideoCard/list-video-card.component.tsx:102]: Currently we don't have access to the video duration. At the time where the duration is implemented we can uncomment this and adapt.

File: apps/polyflix/src/modules/core/components/VideoCard/video-card.component.tsx (hunk 3 above)
Comment (MaloPolese) [reply apps/polyflix/src/modules/core/components/VideoCard/video-card.component.tsx:84]: Currently we don't have access to the video duration. At the time where the duration is implemented we can uncomment this and adapt.

**Comment by MaloPolese** [comment]
> When we are on the homepage, if we scroll down the searchbar isn't reachable anymore and any alternative is available to search content..

I will add an implementation on a second PR

File: apps/polyflix/src/modules/core/components/Jumbotron/jumbotron.component.tsx (hunk 6 above)
Comment (MaloPolese) [reply apps/polyflix/src/modules/core/components/Jumbotron/jumbotron.component.tsx:151]: True, i fix it :up:

File: apps/polyflix/src/modules/core/components/Footer/footer.component.tsx (hunk 1 above)
Comment (MaloPolese) [reply apps/polyflix/src/modules/core/components/Footer/footer.component.tsx:15]: It's juste a placeholder for a second PR

File: apps/polyflix/public/locales/fr/home.json [hunk 7]
```diff
@@ -2,13 +2,12 @@
...
+  "sections": {
     "titles": {
-      "continueWatching": "Continuer à regarder",
-      "latest": "Les plus récentes",
-      "popular": "Populaire",
-      "watchAgain": "Regardez à nouveau",
-      "rated": "Les plus aimées"
+      "popularCourse": "Cours populaires",
+      "lastVideos": "Dernières vidéos",
+      "popularVideos": "Videos populaires",
```
Comment (hugoamalric) [inline apps/polyflix/public/locales/fr/home.json:9]:
```suggestion
      "popularVideos": "Vidéos populaires",
```

File: apps/polyflix/public/locales/fr/home.json [hunk 8]
```diff
@@ -2,13 +2,12 @@
...
+      "mostLikedVideos": "Videos les plus aimées"
```
Comment (hugoamalric) [inline apps/polyflix/public/locales/fr/home.json:10]:
```suggestion
      "mostLikedVideos": "Vidéos les plus aimées"
```

**Review by hugoamalric — CHANGES_REQUESTED** [review]

**Review by LeoFVO — APPROVED** [review]

**Review by kalil-pelissier — APPROVED** [review]

**Review by hugoamalric — APPROVED** [review]

**Pull request merged**